src/module/quickPatrol.js

```diff
@@ -0,0 +1,37 @@
+var mwApi = new mw.Api()
+
+const { _hasRight } = require('./_hasRight')
+
+var checkPatrol = function () {
+  // 没找到检查页面是否需要巡查的 API
+  return _hasRight('patrol')
+}
+
+var quickPatrol = function (revid, done, fail) {
+  var failFallback = function (errorCode, errorThrown) {
+    console.warn('[InPageEdit] 快速巡查失败: ' + errorCode + " " + errorThrown)
+    fail()
+  }
+  console.log('[InPageEdit] 尝试获取巡查 Token...')
+  mwApi.get({
+    action: 'query',
+    meta: 'tokens',
+    type: 'patrol',
+    format: 'json'
+  }).done(function (data) {
```

Comment:
Why not use `mw.user.tokens.get('patrolToken')`?

What's the difference between them?